src/AgentHeaderDescriptor.php

```diff
@@ -52,11 +52,9 @@ class AgentHeaderDescriptor
      *     @type string $phpVersion the PHP version.
      *     @type string $libName the name of the client application.
      *     @type string $libVersion the version of the client application.
-     *     @type string $codeGenName the code generator name of the client library.
-     *     @type string $codeGenVersion the code generator version of the client library.
+     *     @type string $gapicVersion the code generator version of the client library.
```

Comment:
Nit: I'd specify that this is "...the code generator version of the _GAPIC_ library."